backend/middlewares/index.ts

```
@@ -55,11 +57,82 @@ const adminAuth = (req: Request, res: Response, next: NextFunction): void => {
+		return;
+	}
+
+	// First try to verify as admin using JWT
+	try {
+		const secret: string = process.env.JWT_SECRET as string;
+		const decoded = jwt.verify(token, secret) as { role: string };
 
+		// If successful, mark as admin
+		(req as any).authType = 'admin';
+		(req as any).user = decoded;
+		return next();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)